client/cypress/support/commands.js

```diff
@@ -21,3 +21,8 @@ Cypress.Commands.add('getMapCanvas', () => {
   return cy.get('.maplibregl-canvas');
 });
 
+Cypress.Commands.add('waitForMapIdle', (map) => {
+  return new Cypress.Promise((resolve) => {
```

Comment:
It seems odd to return a Promise here?
Maybe it should be "wrapped".
` return cy.wrap(new Cypress.Promise((resolve) ...`

I'm not super familiar with Cypress.

If it compiles and works, probably not super important.

Reply:
Cypress requires Promise-like objects as returns, as it waits for the results to be available - I believe `wrap` could be used by its parent if we needed it, but at this level we need to indicate that its async

Reply:
I missed replying to this item, but it's more a curiosity versus and an actual issue. I was just expecting something like this if a promise was returned (note the `async`).

`Cypress.Commands.add('waitForMapIdle', async (map) => {`